packages/react-storage/src/components/StorageBrowser/subcomponents/PaginationControl/PaginationControl.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { NavElementProps } from '@aws-amplify/ui-react/internal';
+import { useElement } from '../../context/elements';
+import { NextButton } from './NextButton';
+import { PreviousButton } from './PreviousButton';
+import { CurrentPage } from './CurrentPage';
+import { Item } from './Item';
+import { List } from './List';
+
+const PaginationControlPrimitive = <T extends NavElementProps>({
+  ariaLabel = 'Pagination',
+  className: _className,
+  children,
+  ...rest
+}: T): JSX.Element => {
+  const Nav = useElement('Nav');
+  const baseClassName = 'storage-browser-pagination-control';
+  const className = _className ?? baseClassName;
+
+  return (
+    <Nav {...rest} aria-label={ariaLabel} className={className}>
+      {children}
+    </Nav>
+  );
+};
+
+const PaginationControl = Object.assign(PaginationControlPrimitive, {
```

Comment:
@calebpollman Do you have a preference on how we export these subcomponents that have subcomponents? In this case the usage of PaginationControl is something like:
```
<PaginationControl>
  <PaginationControl.List>
    <PaginationControl.Item>
      <PaginationControl.PreviousButton />
       // etc. etc.
 ```

so I just have this basic Object.assign

 ```
 const PaginationControl = Object.assign(PaginationControlPrimitive, {
  Item,
  List,
  NextButton,
  CurrentPage,
  PreviousButton,
});

export { PaginationControl };
```